Write upgrade instructions for osg-configure v4 (#786)
opensciencegrid/docs: docs/release/updating-to-osg-36.md

AI (Carl): https://opensciencegrid.atlassian.net/browse/SOFTWARE-4473

#### OSG-Configure ####

The OSG 3.6 release series contains OSG-Configure 4, a major version upgrade from the previously released versions in the OSG.
See the [OSG-Configure 4.0.0 release notes](https://github.com/opensciencegrid/osg-configure/releases/tag/v4.0.0)
for an overview of the changes.
Several configuration modules and options were removed or deprecated and CE configuration has been simplified;
the update from version 3 to version 4 will require some manual changes to your configuration.

To update OSG-Configure, perform the following steps:

1.  Update the osg-configure packages:

        :::console

>         root@host# yum update 'osg-configure*'

Reviewer: You can drop this step as I plan on having a catch-all RPM update section (https://github.com/opensciencegrid/docs/pull/786/files#diff-56ab23f60058662c854a569e806dba7443ce99223a41cbb317825952fb6cf920R14)

> 2.  Merge any `*.rpmnew` files in `/etc/osg/config.d/`

Reviewer: ```suggestion
1.  Merge any `*.rpmnew` files in `/etc/osg/config.d/`
```

Just to make diffs easier if we plan on adding, removing, or reordering steps

> 3.  Uninstall `osg-configure-gip` and `osg-configure-misc` if they are installed:

Reviewer: ```suggestion
1.  Uninstall `osg-configure-gip` and `osg-configure-misc` if they are installed:
```

        :::console
        root@host# yum erase osg-configure-gip osg-configure-misc

> 4.  If `/etc/osg/config.d/30-gip.ini.rpmsave` exists, merge its contents into `31-cluster.ini`

Reviewer: ```suggestion
1.  If `/etc/osg/config.d/30-gip.ini.rpmsave` exists, merge its contents into `31-cluster.ini`
```

> 5.  If you are on a CE, edit the `Site Information` configuration section (in `40-siteinfo.ini`) and add:

Reviewer: ```suggestion
1. Edit the `Site Information` configuration section (in `40-siteinfo.ini`) and add:
```

`osg-configure` in 3.6 is just a CE package, at least for now. We should also tell users to only add this if they don't have it already set.

        resource_group = <TOPOLOGY RESOURCE GROUP FOR THIS HOST>

> 6.  Run osg-configure to apply your changes:

Reviewer: ```suggestion
1.  Run osg-configure to apply your changes:
```
